conf/spark-env.sh

```diff
@@ -53,17 +59,28 @@ if ls ${MESOS_SANDBOX}/*.base64 1> /dev/null 2>&1; then
 fi
 
 if [[ -n "${KRB5_CONFIG_BASE64}" ]]; then
-    if base64 --help | grep -q GNU; then
-          BASE64_D="base64 -d" # GNU
-      else
-          BASE64_D="base64 -D" # BSD
-    fi
     echo "spark-env: Copying krb config from $KRB5_CONFIG_BASE64 to /etc/" >&2
     echo "${KRB5_CONFIG_BASE64}" | ${BASE64_D} > /etc/krb5.conf
 else
     echo "spark-env: No kerberos KDC config found" >&2
 fi
 
+if [[ -n "${KEYSTORE_BASE64}" ]]; then
+    echo "spark-env: Copying keystore to SANDBOX" >&2
+    echo "${KEYSTORE_BASE64}" | ${BASE64_D} > ${MESOS_SANDBOX}/server.jks
```

Comment:
I'd suggest using file based secrets that are more secure than writing contents of keystore to mesos sandbox.